src/components/line/LineFigures.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+import { reactive, onMounted } from 'vue'
+
+import { apiClientService } from '@/services/api.client'
+import UiNetworkFigure from '../ui/UiNetworkFigure.vue'
+import UiVerticalSeparator from '../ui/UiVerticalSeparator.vue'
+import type { LineFigureModel } from '../../model/line-figures.model'
+
+const state = reactive({
+  lineFigures: null as null | LineFigureModel[],
+})
+
+onMounted(async () => {
+  state.lineFigures = await apiClientService.fetchLineFigure()
+})
+
+function getLength(networkFigures: null | LineFigureModel[]): Number {
+  if (networkFigures == null) {
+    return 0
+  } else {
+    return networkFigures.length
+  }
+}
+</script>
+
+<template>
+  <div
+    class="flex flex-row px-4 py-3 gap-3 bg-slate-50 border rounded-lg border-slate-100"
+  >
+    <template
```

Comment:
why do you use a extra `template` here ?

Reply:
I think that each block should be an `<article>`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

Reply:
@fgravin because of this https://vuejs.org/guide/essentials/list.html#v-for-with-v-if I need to use a template so that I can use v-if on v-for.